### ISSUE TITLE
[Request] See and autofill events

### ISSUE THREAD

**Issue by Strauman** [robertklep/name.klep.flow-event-bus#2]
Hi!
Thanks for this amazing app.
I do have a wish though. In the CountDown app ([source](https://github.com/NotQuiteZen/homey.countdown), [App](https://homey.app/no-no/app/homey.countdown/Countdown-timers/)) it's possible to see all created count downs, and autofill them.

Would this be possible in Event Bus too?

Thanks again

Andreas

**Comment by robertklep** [comment]
No, that's not possible, because (unlike the Countdown app) the app doesn't maintain a list of events internally. An app also isn't aware which flows will use it, so it cannot find out (up front) which of your flows emit which event, and which flows react to which event.
